src/libs/OptionsListUtils.ts

```diff
@@ -1981,6 +1990,9 @@ function getOptions(
                 optionsToExclude.push({login: reportOption.login});
             }
         }
+    } else if (recentAttendees && recentAttendees?.length > 0) {
+        recentAttendees.filter((attendee) => attendee.login ?? attendee.displayName).forEach((a) => optionsToExclude.push({login: a.login ?? a.displayName}));
+        recentReportOptions = recentAttendees.map((attendee) => attendee as ReportUtils.OptionData);
```

Comment:
It seems this `map` does nothing, why do we need an assertion here?

Reply:
Yes, map is useless, assertion is needed because `reportID` is required in OptionData and is optional in Attendee type

Reply:
How about this?
```suggestion
        recentReportOptions = recentAttendees as ReportUtils.OptionData[];
```

Reply:
Yes, it works, I thought I've pushed it